test/langtools/tools/javac/warnings/Serial/EnumSerial.java

```diff
@@ -2,7 +2,7 @@
  * @test /nodynamiccopyright/
  * @bug 8202056
  * @compile/ref=EnumSerial.out -XDrawDiagnostics -Xlint:serial EnumSerial.java
- * @compile/ref=empty.out -XDrawDiagnostics               EnumSerial.java
+ * @compile/ref=empty.out      -XDrawDiagnostics               EnumSerial.java
```

Comment:
extra spaces snuck in

Reply:
> extra spaces snuck in

I added the extra spaces to align the argument on the second `@compile` line with the corresponding arguments on the first line.